public/Compute_TWI/compute_twi.py

```diff
@@ -0,0 +1,26 @@
+@fused.udf
+def udf(bbox):
+    import wbt
```

Comment:
```suggestion
def udf(bbox, out_tif_name=out_tif_name, wbt_args: dict = wbt_args, min_max=min_max):
    import wbt
    wbt_args = json.loads(wbt_args) if isinstance(wbt_args, str) else wbt_args
```

For flexibility, would it make sense to pass these as parameters to the UDF?

Reply:
If you think it makes sense, I'm ok with it.